### PR TITLE
Nerf beams against ridley

### PR DESCRIPTION
**Pull request by ThanatosGit**
Don't know when or if I will test it. But we have beta testers :)
(raising it for wave beam and plasma beam was working)

### CODE REVIEW

**Review by dyceron — COMMENTED** [review]
Do we not have worry about charge? Also what about ice? It ends up being stronger than wave and power now

**Comment by ThanatosGit** [comment]
> Do we not have worry about charge?
See:
> Don't know when or if I will test it. But we have beta testers :)

All bullets should be:
Power Beam: 3 (25 * 0,12)
Ice: 10
Wave: 6 (50 *0,12)
Spazer: 25,2 (210 * 0,12)
Plasma: 36 (default of 12 * 3)

So, we make Wave to be 14 or 15?

**Comment by dyceron** [comment]
> All bullets should be:
> Power Beam: 3 (25 * 0,12)
> Ice: 10
> Wave: 6 (50 *0,12)
> Spazer: 25,2 (210 * 0,12)
> Plasma: 36 (default of 12 * 3)
> 
> So, we make Wave to be 14 or 15?
> 

We could have Wave be 0.17 which is 8.5 damage or 1/3 of spazer (literally makes sense). Following that, Power could be 4.25 (0.17) or half of Wave as its normally half the strength of Wave, then have Ice use 0.33 or 3.33 damage

File: src/open_samus_returns_rando/specific_patches/static_fixes.py [hunk 1]
```diff
@@ -79,10 +79,18 @@ def shoot_supers_without_missiles(editor: PatcherEditor):
     gun[20]["params"]["Param5"]["value"] = ""
 
 
+def nerf_ridley_fight(editor: PatcherEditor):
+    ridley_bmsad = editor.get_file("actors/characters/ridley/charclasses/ridley.bmsad", Bmsad)
+    ridley_bmsad.components["LIFE"].raw["fields"]["fSpazerBeamFactor"]["value"] = 0.12
+    ridley_bmsad.components["LIFE"].raw["fields"]["fWaveBeamFactor"]["value"] = 0.12
```
Comment (dyceron) [inline src/open_samus_returns_rando/specific_patches/static_fixes.py:85]: I think we decided on 14? Change it 0.28.

**Review by dyceron — CHANGES_REQUESTED** [review]

**Review by dyceron — APPROVED** [review]

**Pull request merged**